executioncontext/projectview/README.md

```diff
@@ -297,3 +297,8 @@ _We are working on it, you can expect support for this section in future release
 _We are working on it, you can expect support for this section in future releases._
 
 ---
+
+#### build_manual_targets
+
+Targets with ["manual"] tag are excluded in wildcard target patterns and not build, run or tested unless explicitly
```

Comment:
```suggestion
A flag specifying if targets with `manual` tag should be built.

Flag is boolean value, so it can take either true or false. In the first case targets with `manual` tag will be build, otherwise they won't.

##### example:

```
build_manual_targets: true
```

##### default:

```
build_manual_targets: false
```
```